src/Entity/PropertyId.php

```diff
@@ -64,7 +70,7 @@ public function getEntityType() {
 	 * @return string
 	 */
 	public function serialize() {
-		return json_encode( array( 'property', $this->serialization ) );
+		return json_encode( array( 'property', $this->getSerialization() ) );
```

Comment:
The reason is that `serialization` is supposed to be private to the parent class. The only thing blocking that is that it's still set directly in the `unserialize` method.